test/lightning_web/live/workflow_live/index_test.exs

```diff
@@ -17,6 +18,42 @@ defmodule LightningWeb.WorkflowLive.IndexTest do
              |> element("#workflows-#{project.id}", "No workflows yet")
     end
 
+    test "only users with MFA enabled can access workflows for a project with MFA requirement",
+         %{
+           conn: conn
+         } do
+      user = user_with_mfa_fixture()
```

Comment:
Could you move the new `user_with_mfa_fixture` into the Factories module and rewrite it to conform with ExMachinas' API.